addons/website/static/src/components/dialog/seo.js

```diff
@@ -465,7 +465,7 @@ export class OptimizeSEODialog extends Component {
                 'website_id': this.website.currentWebsite.id,
             },
         });
-        this.website.goToWebsite({path: this.url.replace(this.previousSeoName || this.seoNameDefault, seoContext.seoName)});
+        this.website.goToWebsite({path: this.url.replace(this.previousSeoName || this.seoNameDefault, seoContext.seoName || this.seoNameDefault)});
```

Comment:
The change should be in the `/slides` controller, which is the root cause, this is just one of the possibly multiple cases highlighting the bug in the `/slides` controller.